### PR TITLE
Made SNMP privacy protocol selectable

### PR DESCRIPTION
**Pull request by dommi22m**
This is my version to make the privacy protocol selectable. AES128 is used as default.

### CODE REVIEW

**Review by wernerfred — CHANGES_REQUESTED** [review]
LGTM 👍🏻 

Please add the new option `[-s [AES128|DES]]` to the usage in the `README.md` accordingly.

Why specifically did you choose `-s`? Maybe we should change this to `-e`(encryption?) to save `-s` for future usecases that are more related to the letter `s`? What's your opinion on that?

**Comment by dommi22m** [comment]
> Please add the new option [-s [AES128|DES]] to the usage in the README.md accordingly.
Done!

> Why specifically did you choose -s? Maybe we should change this to -e(encryption?) to save -s for future usecases that are more related to the letter s? What's your opinion on that?
Yes you are right! I was thinking in the direction of "secret". But that would not be logical and also wrong in terms of content.

File: README.md [hunk 1]
```diff
@@ -34,10 +34,11 @@ check_synology --help
 
 ```shell
 check_synology
-usage: check_synology [-h] [-w W] [-c C] [-p PORT] hostname username authkey privkey {load,memory,disk,storage,update,status}
+usage: check_synology [-h] [-w W] [-c C] [-p PORT] [-e PROTOCOL ] hostname username authkey privkey {load,memory,disk,storage,update,status}
```
Comment (wernerfred) [inline README.md:37]:
```suggestion
usage: check_synology [-h] [-w W] [-c C] [-p PORT] [-e PROTOCOL] hostname username authkey privkey {load,memory,disk,storage,update,status}
```

**Review by wernerfred — CHANGES_REQUESTED** [review]
Small whitespace fix, rest LGTM 👍🏻

**Comment by wernerfred** [comment]
@all-contributors add @dommi22m for code

**Review by wernerfred — APPROVED** [review]

**Pull request merged**